src/i18n.tsx

```diff
@@ -24,11 +35,17 @@ export async function dynamicActivate(locale: string) {
 }
 
 export function LanguageProvider({ children }: { children: ReactNode }) {
```

Comment:
ok how about in the redux store we store `undefined` or `selectedLocale` and if selectedLocale is undefined or not present in the locale enum values we fallback to detecting the locale